jsonschema-validation.xml

```diff
@@ -786,25 +792,33 @@
 
             <section title="contentMediaType">
                 <t>
-                    The value of this property must be a media type, as defined by
-                    <xref target="RFC2046">RFC 2046</xref>. This property defines the media
-                    type of instances which this schema defines.
+                    If the instance is a string, this property defines the media type
+                    of the contents of the string.  If "contentEncoding" is present,
+                    this property describes the decoded string.
                 </t>
-
                 <t>
-                    The value of this property MUST be a string.
+                    The value of this property MUST be a string, which MUST be a media type,
+                    as defined by <xref target="RFC2046">RFC 2046</xref>.
```

Comment:
See the [Implementation Requirements](https://github.com/json-schema-org/json-schema-spec/pull/673/files/d91706a702ac62ce11af3ce729dc5275f3c209b0#diff-2391a2ef934e9aed11cbf87a65eb61c9R743) section.  The short version is that you aren't expected to be ably to identify or validate _any_ of them.  The slightly longer version is that it's basically like `format`, except we are not picking a default set of recognizable values.

I kind of expect most implementations to handle these as annotations passed on to the application, maybe with a callback mechanism to automatically validate the decoded text against `contentSchema`.

Since this is not changed in this PR (I just moved the phrase "media type" from the first paragraph to the second), I'm not going to hold up this PR on it.  Although if you think there is something specific to `contentSchema` (as opposed to `contentMediaType`) that needs clarification I would be happy to work on that a bit more.

Reply:
Hmm... I could see that people might expect `contentSchema` to always be implemented, without really thinking about the dependency on having the right plugin (or whatever) for the underlying `contentMediaType` and `contentEncoding` values.  Is that your concern, @gregsdennis ?

Reply:
My concern is that the "MUST" implies that implementations are required to be able to validate that a given value is a media type as defined by RFC 2046.

If it's like `format`, then maybe similar language can be used that allows implementations to defer that validation to the application in the form of returning the value as an annotation, as you said.

Reply:
@gregsdennis the "Implementation Requirements" section is practically copy-pasted from the `format` section.  What is missing?  I don't want to repeat them for each keyword, if you read the spec outside of the diff lines I think it's already clear.  Can you point to specific problems in that section?

Also, I don't hear anything specific to `contentSchema` here (you're talking about moved text, not new requirements) so please file it as a new issue.  This PR is just for `contentSchema`.